src/solver/optim-model-filler/ComponentFiller.cpp

```diff
@@ -421,21 +478,23 @@ void ComponentFiller::addObjectives(const LinearProblemApi::FillContext& ctx)
           objective.expression().RootNode());
 
         auto& pb = optimEntityContainer_.Problem();
+        std::optional<double> objectiveOffset;
         for (const auto& expr: linearExpression)
         {
             for (const auto& [index, value]: expr)
             {
                 pb.setObjectiveCoefficient(solverVariables[static_cast<std::size_t>(index)].get(),
                                            value);
             }
+            objectiveOffset = updateOffset(objectiveOffset, expr.constant(), objective.Id(), pb);
```

Comment:
I understand that here, we want to add the offset only once, whatever the time dimension of the offset.
More precisely, I see an objective as something of the form : A.x + B on the time interval {t1, ..., tn}, where 
- A is a matrix of coefs, 
- x i= {x[t0], x[t1], ...,x[tn]}
- B = {B[t0], B[t1], ...,B[tn]}